lib/stages/visTestPattern.cpp

```diff
@@ -238,6 +238,7 @@ void visTestPattern::main_thread() {
                     avg_err /= (float)num_bad;
 
                     // write frame report to outfile
+                    outfile.imbue(std::locale::classic());
```

Comment:
So do we still need this?  I guess cannot hurt having it.

Reply:
Jim's opinion was to keep it, just to be explicit